src/runtimes/node/bundlers/nft/index.ts

```diff
@@ -34,6 +36,8 @@ const bundle: BundleFunction = async ({
     includedFilesBasePath || basePath,
   )
   const {
+    aliases,
+    mainFile: normalizedMainFile,
```

Comment:
can you elaborate on what `normalizedMainFile` is? Is it a normalized path to the bundle entry file?

Reply:
The `mainFile` property refers to both the path of a function's entry file, as found in a Netlify project, and to the path of the entry file inside the generated bundle.

Most times this file name is the same, because we find `function.js` and generate `function.js`. But when transpiling, we need to rewrite `function.ts` as `function.js`, and that's what `normalizedMainFile` refers to.

Although we use it in a bunch of places, I realise it's not a great name, so perhaps we could make that a bit clearer. Do you have any suggestions?